solr/solr-ref-guide/src/collections-api.adoc

```diff
@@ -750,36 +750,38 @@ http://localhost:8983/solr/admin/collections?action=LISTALIASES&wt=xml
 ----
 
 [[modifyalias]]
```

Comment:
"modifyalias" here is now incorrect; should be "ALIASPROP".  This would affect some of the examples below too.